test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -146,7 +147,7 @@ public void EarlyStoppingTest()
         /// <summary>
         /// Multiclass Logistic Regression test.
         /// </summary>
-        [Fact]
+        [NotAppleSiliconFactAttribute("Temporarily skipping while helix issues are resolved. Tracked in issue #5845")]
```

Comment:
These issue numbers should be full links
`[https://github.com/dotnet/machinelearning/issues/5845`](https://github.com/dotnet/machinelearning/issues/5845`)

Reply:
Fixed.